### PR TITLE
Slow Page Transitions and Infinite Loading with Internal Navigation Using Link Component in Next.js 14.2

### PR DESCRIPTION
**Pull request by gimenete**

### CODE REVIEW

**Comment by gimenete** [comment]
a

**Comment by gimenete** [comment]
Close it!

**Comment by gimenete** [comment]
reopen!

**Comment by gimenete** [comment]
close!

**Comment by gimenete** [comment]
Reopen!

**Review by gimenete — COMMENTED** [review]
Here goes a comment

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 asdf
+asasdfasfd
```
Comment (gimenete-stripe) [inline README.md:2]: This is a comment

Comment (gimenete) [reply README.md:2]: Good catch!

Comment (gimenete) [reply README.md:2]: nice!

**Review by gimenete-stripe — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
 asdf
```
Comment (gimenete-stripe) [inline README.md:1]: foo

**Review by gimenete-stripe — COMMENTED** [review]
a comment

**Comment by gimenete** [comment]
hello!

**Pull request merged**